Vungle/src/main/java/com/mopub/mobileads/VungleBanner.java

```diff
@@ -122,22 +129,44 @@ public void run() {
             return;
         }
 
-        //currently we only support MREC for banners. This will require to be reworked once we add other sizes
-        if (adConfig.getAdSize() != AdConfig.AdSize.VUNGLE_MREC) {
+        adConfig.setAdSize(vungleAdSize);
+
+        sVungleRouter.addRouterListener(mPlacementId, mVungleRouterListener);
+
+        VungleMediationConfiguration.adConfigWithLocalExtras(adConfig, localExtras);
+        if (VungleMediationConfiguration.isStartMutedNotConfigured(localExtras)) {
+            adConfig.setMuted(true); // start muted by default
+        }
+
+        if (AdSize.isBannerAdSize(vungleAdSize)) {
+            if (sVungleRouter.isBannerAdPlayable(mPlacementId, vungleAdSize)) {
+                mVungleRouterListener.onAdAvailabilityUpdate(mPlacementId, true);
+                MoPubLog.log(mPlacementId, LOAD_SUCCESS, ADAPTER_NAME);
+            } else {
+                sVungleRouter.loadBannerAd(mPlacementId, vungleAdSize, mVungleRouterListener);
+                MoPubLog.log(mPlacementId, LOAD_ATTEMPTED, ADAPTER_NAME);
+            }
+        } else if (VUNGLE_MREC == getVungleAdSize(localExtras)) {
```

Comment:
Here we don't need to fire `getVungleAdSize()` again. We can use `vungleAdSize` variable directly.